sdmetrics/reports/multi_table/_properties/inter_table_trends.py

```diff
@@ -130,12 +130,13 @@ def _generate_details(self, real_data, synthetic_data, metadata, progress_bar=No
                 denormalized_real, denormalized_synthetic, merged_metadata,
                 progress_bar=progress_bar, column_pairs=parent_child_pairs
             )
+
             details['Parent Table'] = parent
             details['Child Table'] = child
             details['Foreign Key'] = foreign_key
-            details['Column 1'] = details['Column 1'].str.replace(f'{parent}.', '', n=1)
-            details['Column 2'] = details['Column 2'].str.replace(f'{child}.', '', n=1)
-
+            if not details.empty:
+                details['Column 1'] = details['Column 1'].str.replace(f'{parent}.', '', n=1)
+                details['Column 2'] = details['Column 2'].str.replace(f'{child}.', '', n=1)
```

Comment:
nice! Can we add a unit test for this? Additionally, is it possible that 'Column 1' or 'Column 2' will ever be missing?

Reply:
Added a unit test! No it is impossible for those columns to be missing as the `_generate_details` in `column_pair_trends.py` will always append those columns.